tests/test_deployment.py

```
@@ -202,6 +204,13 @@ def test_setup_excludes_sensitive_config(self):
         with raises(configparser.NoOptionError):
             deploy_config.get('Parameters', 'something_sensitive')
 
+    def test_reraises_db_connection_error(self, subject):
+        with mock.patch('dallinger.deployment.db.check_connection') as checker:
+            checker.side_effect = Exception("Boom!")
```

I think this should raise the postgres error we expect, not bare `Exception`.